floodviz/static/js/floodviz_onready.js

```diff
@@ -0,0 +1,28 @@
+document.addEventListener("DOMContentLoaded", function (event) {
+
+	"use strict";
+
+	// Map figure 
+	var map_options = { 'height': FV.mapinfo.height,
+						'width': FV.mapinfo.width,
```

Comment:
This is somewhere where an IDE or other tool might help with formatting. I'd rather see this like
```
map_options = {
     'height': FV.mapinfo.height,
    'width': FV.mapinfo.width
}
```
Don't need to fix now but you should look for tools that will help with this kind of formatting.